### PR TITLE
[update] added reconstructed-image-data-schema

### PR DESCRIPTION
**Pull request by i-z-j**

### CODE REVIEW

File: reconstructed-image-data-schema.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Reconstructed Image Data Schema
+
+## Version
+1.1.0
```
Comment (bewantbe) [inline reconstructed-image-data-schema.md:4]: Where is 1.0 ?

**Review by bewantbe — COMMENTED** [review]
Can user (and reader code) have a way to easily distinguish VISoR Raw, Reconstructed slice and Reconstructed brain?

File: reconstructed-image-data-schema.md (hunk 1 above)
Comment (i-z-j) [reply reconstructed-image-data-schema.md:4]: release tag is at repo level, thus the version of Reconstructed Image Data Schema follows the existing ones.

**Comment by i-z-j** [comment]
> Can user (and reader code) have a way to easily distinguish VISoR Raw, Reconstructed slice and Reconstructed brain?

good question, currently we use top-level data directory path (VISoR_Data vs VISoR_Reconstruction) to diff them, but IMO, it might be better to have each sample data directory contains both raw and reconstructed images.

**Pull request closed without merging**